lib/models/asyncapi.js

```diff
@@ -268,18 +268,31 @@ function recursiveSchema(schema, callback) {
   if (schema.type() !== undefined) {
     switch (schema.type()) {
       case 'object':
-        const props = schema.properties();
-        for (const [, propertySchema] of Object.entries(props)) {
-          recursiveSchema(propertySchema, callback);
+        if(schema.additionalProperties() !== undefined && typeof schema.additionalProperties() !== "boolean"){
+          var additionalSchema = schema.additionalProperties()
```

Comment:
Always prefer `let` or `const`. The scope of `var`s is the whole function which may lead to undesired behaviors in the future when someone "touches" this code.

```suggestion
          const additionalSchema = schema.additionalProperties()
```